src/pages/core/guides/create-soulbound-nft-asset.md

```diff
@@ -0,0 +1,501 @@
+---
+title: Soulbound Assets in MPL Core
+metaTitle: Soulbound Assets in MPL Core | Core Guides
+description: This Guide explores the different options for soulbound Assets in MPL Core
+---
+
+
+Soulbound NFTs are non-fungible tokens that are permanently bound to a specific wallet address and cannot be transferred to another owner. They are useful for representing achievements, credentials, or memberships that should remain tied to a specific identity.  {% .lead %}
+
+## Overview
+
+In this guide, we'll explore how to create soulbound assets using MPL Core and the Umi Framework. Whether you're a developer looking to implement soulbound NFTs in TypeScript or just want to understand how they work, we'll cover everything from basic concepts to practical implementation. We'll examine different approaches for making assets soulbound and walk through creating your first soulbound NFT within a collection.
+
+In MPL Core, there are two main approaches to create soulbound NFTs:
+
+### 1. Permanent Freeze Delegate Plugin
+- Makes assets completely non-transferrable and non-burnable
+- Can be applied at either:
+  - Individual asset level
+  - Collection level (more rent efficient)
+- Collection-level implementation allows thawing all assets in a single transaction
+
+### 2. Oracle Plugin
+- Makes assets non-transferrable but still burnable
+- Can also be applied at:
+  - Individual asset level  
+  - Collection level (more rent efficient)
+- Collection-level implementation allows thawing all assets in a single transaction
+
+## Creating Soulbound NFTs with the Permanent Freeze Delegate Plugin
+
+The Permanent Freeze Delegate Plugin provides functionality to make assets non-transferrable by freezing them. When creating a soulbound asset, you would:
+
+1. Include the Permanent Freeze plugin during asset creation
+2. Set the initial state to frozen
+3. Set the authority to None, making the frozen state permanent and immutable
+
+This effectively creates a permanently soulbound asset that cannot be transferred or thawed.
```

Comment:
[This Should be a suggestion but with the code ticks it doesn't work]

Just like this:
```javascript
...plugins: [
      {
        type: 'PermanentFreezeDelegate',
        frozen: true,
        authority: { type: "None" },
      },
    ],
```

This effectively creates a permanently soulbound asset that cannot be transferred or thawed.

Reply:
Added it, thank you.